File: nvxapp.server.service/ClientServer-Service/GestionePresenze/Dip_CompetenzaService/Models/Dip_CompetenzaModel.cs
using nvxapp.server.service.ClientServer_Service.ModelsBase;

namespace nvxapp.server.service.ClientServer_Service.GestionePresenze.Dip_CompetenzaService.Models
{
    public class Dip_CompetenzaModel
    {
        public int Id { get; set; }
        public int IdDip_Anagrafica { get; set; }
        public int IdAz_Competenza { get; set; }
    }

    public class Dip_Competenza_GetAll_InModel { }

    public class Dip_Competenza_GetAll_OutModel : ModelResult
    {
        public List<Dip_CompetenzaModel> Dip_Competenza { get; set; } = new List<Dip_CompetenzaModel>();
    }
}

File: nvxapp.client/src/nvx/ClientServer-Service/GestionePresenze/Dip_Competenza/Models/dip-competenza-model.ts
import { ModelResult } from "../../../ModelsBase/model-result";

export class Dip_CompetenzaModel {
  public id!: number;
  public idDip_Anagrafica!: number;
  public idAz_Competenza!: number;
}

export class Dip_Competenza_GetAll_InModel {}

export class Dip_Competenza_GetAll_OutModel extends ModelResult {
  public dip_Competenza: Dip_CompetenzaModel[] = [];
}
